packages/service-content/src/lib/blogs/import/main.ts

```diff
@@ -17,9 +17,9 @@ export const createProcessMainFile = (transaction: TransactionSql) => {
     if (!file) return;
 
     if (file.kind === 'removed') {
+      console.log('No file provided, exiting function early.');
```

Comment:
No need for console.log, as in all other import files.
This is mainly just usefull for typescript, because removed files have another type.